introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit soluta, atque repellendus consequuntur at minima ex vitae unde tempora harum sapiente rerum eos deleniti ullam obcaecati, laudantium libero dolore. Saepe?
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.